.gitignore

```diff
@@ -1,15 +1,14 @@
 # environment variables
-environment*
+/environment*
 
 # auspice build
-auspice-client/index.html
-auspice-client/dist/
+/auspice-client/index.html
+/auspice-client/dist/
 
 # misc #
-.DS_Store
-node_modules/
-npm-debug.log
-scratch
+/.DS_Store
+/node_modules/
```

Comment:
`.DS_Store` wants to be unanchored since it's per-directory (IIRC, present in any directory browsed by the Finder on macOS, or maybe browsed + custom view attrs).

`/node_modules/` will want to expand to two lines to also cover `/static-site/node_modules/`.

Reply:
Done: 68a2765ac37c129f526c58ceb951a5cd6aaf091a